src/views/ModelPlan/TaskList/Basics/Overview/index.tsx

```diff
@@ -0,0 +1,333 @@
+import { Trans, useTranslation } from 'react-i18next';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+  Button,
+  Fieldset,
+  IconAdd,
+  IconArrowBack,
+  Label,
+  Radio
+} from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageNumber from 'components/PageNumber';
+import AutoSave from 'components/shared/AutoSave';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import TextAreaField from 'components/shared/TextAreaField';
+import GetModelPlanQuery from 'queries/GetModelPlanQuery';
+import {
+  GetModelPlan,
+  GetModelPlanVariables
+} from 'queries/types/GetModelPlan';
+import { UpdatePlanBasics as UpdatePlanBasicsType } from 'queries/types/UpdatePlanBasics';
+import UpdatePlanBasics from 'queries/UpdatePlanBasics';
+import flattenErrors from 'utils/flattenErrors';
+import planBasicsSchema from 'validations/planBasics';
+
+interface PlanBasicsOverviewTypes {
+  modelType: string;
+  problem: string;
+  goal: string;
+  testInterventions: string;
+  note: string;
+}
+
+const Overview = () => {
+  const { t } = useTranslation('basics');
+  const { t: h } = useTranslation('draftModelPlan');
+  const { modelId } = useParams<{ modelId: string }>();
+
+  const formikRef = useRef<FormikProps<PlanBasicsOverviewTypes>>(null);
+  const history = useHistory();
+  const [hasAdditionalNote, setHasAdditionalNote] = useState(false);
+
+  const { data } = useQuery<GetModelPlan, GetModelPlanVariables>(
+    GetModelPlanQuery,
+    {
+      variables: {
+        id: modelId
+      }
+    }
+  );
+
+  const { modelName } = data?.modelPlan || {};
+  const { id, modelType, problem, goal, testInventions, note } =
+    data?.modelPlan?.basics || {};
+
+  const [update] = useMutation<UpdatePlanBasicsType>(UpdatePlanBasics);
+
+  const initialValues = {
+    modelType: modelType ?? '',
+    problem: problem ?? '',
+    goal: goal ?? '',
+    testInterventions: testInventions ?? '',
+    note: note ?? ''
+  };
+
+  const handleFormSubmit = (
+    formikValues: PlanBasicsOverviewTypes,
+    isAutoSave: boolean = false,
+    continuesToNextStep: boolean = true
+  ) => {
+    update({
+      variables: {
+        input: {
+          id,
+          modelPlanID: modelId,
+          modelType: formikValues.modelType,
+          problem: formikValues.problem,
+          goal: formikValues.goal,
+          testInventions: formikValues.testInterventions,
+          note: formikValues.note
+        }
+      }
+    })
+      .then(response => {
+        if (!response?.errors) {
+          if (continuesToNextStep) {
```

Comment:
On a successful response the redirect prop would trigger a conditional like this

```      
          if (redirect === 'next') {
            history.push('/models/${modelId}/task-list/basics/milestones');
          } else if (redirect === 'back') {
            history.push('/models/${modelId}/task-list/basics');
          } else if (redirect === 'task-list') {
            history.push('/models/${modelId}/task-list/');
          }
```